fuel-tx/Cargo.toml

```
@@ -23,9 +23,9 @@ serde = { version = "1.0", default-features = false, features = ["alloc", "deriv
 serde_json = { version = "1.0", default-features = false, features = ["alloc"], optional = true }
 
 [dev-dependencies]
-bincode = { version = "1.3", default-features = false }
+bincode = { workspace = true }
 fuel-crypto = { workspace = true, default-features = false, features = ["random"] }
-fuel-tx = { path = ".", features = ["builder", "random"] }
+fuel-tx = { path = ".", features = ["builder", "random", "serde"] }
```

Also should only be enabled when the serde feature is turned on

```suggestion
fuel-tx = { path = ".", features = ["builder", "random"] }
```

---

Added
```rust
#[cfg(feature = "serde")]
mod bytes;
#[cfg(feature = "serde")]
mod display;
#[cfg(not(feature = "serde"))]
use bincode as _;
```